### PR TITLE
Add info logging for Job failure and cancellation

### PR DESCRIPTION
**Pull request by bradyoo**

### CODE REVIEW

**Comment by bradyoo** [comment]
18:15:59.266 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-install-plugin:2.4:install (default-install) on project alluxio-core-common: Failed to install metadata org.alluxio:alluxio-core-common:2.3.0-SNAPSHOT/maven-metadata.xml: Could not parse metadata /home/jenkins/.m2/repository/org/alluxio/alluxio-core-common/2.3.0-SNAPSHOT/maven-metadata-local.xml: in epilog non whitespace content is not allowed but got a (position: END_TAG seen ...</metadata>\na... @21:2)  -> [Help 1]
18:15:59.267 [ERROR] 

????

**Comment by bradyoo** [comment]
alluxio-bot, test this please.

**Comment by bradyoo** [comment]
@gpang

File: job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java [hunk 1]
```diff
@@ -250,10 +250,14 @@ private synchronized void updateStatus() {
           mPlanInfo.setStatus(Status.FAILED);
           if (mPlanInfo.getErrorMessage().isEmpty()) {
             mPlanInfo.setErrorMessage("Task execution failed: " + info.getErrorMessage());
+            LOG.info("Job failed Id={} Error={}", mPlanInfo.getId(), info.getErrorMessage());
```
Comment (gpang) [inline job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java:253]: should this be `LOG.error`? Also, is there something in `TaskInfo info` that would be useful to print to the log?

File: job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java [hunk 2]
```diff
@@ -250,10 +250,14 @@ private synchronized void updateStatus() {
...
           }
           return;
         case CANCELED:
           if (mPlanInfo.getStatus() != Status.FAILED) {
+            if (mPlanInfo.getStatus() != Status.CANCELED) {
+              LOG.info("Job cancelled Id={}", mPlanInfo.getId());
```
Comment (gpang) [inline job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java:259]: Maybe this should be `warn`? Also, is there something in `TaskInfo info` that would be useful to print to the log?

File: job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java (hunk 1 above)
Comment (bradyoo) [reply job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java:253]: There is a LOG.info of when the job starts that contains more details. Job failing is actually fairly frequent enough that info log is okay.

File: job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java (hunk 2 above)
Comment (bradyoo) [reply job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java:259]: I also think info here is fine. Also we have a Job Start info log that lets you know about some of the details.

**Comment by bradyoo** [comment]
Responded.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by bradyoo** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Comment by bradyoo** [comment]
@gpang: Can you cherry-pick this to branch-2.2 for me please?

**Comment by bradyoo** [comment]
@gpang can you cherry-pick this to branch-2.2 for me please?

**Comment by bradyoo** [comment]
Ping?

**Comment by bradyoo** [comment]
@gpang

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**